tools/install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+DIR="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+WORKSPACE_ROOT=$DIR/..
+pushd $WORKSPACE_ROOT
+OPENTELEMETRY_INSTALL_DIR=${1-/usr/local}
+echo "Install SDK to $OPENTELEMETRY_INSTALL_DIR"
+mkdir -p $OPENTELEMETRY_INSTALL_DIR/lib
+# TODO: install libraries?
+cp -R api/include $OPENTELEMETRY_INSTALL_DIR
+cp -R sdk/include/opentelemetry/sdk $OPENTELEMETRY_INSTALL_DIR/include/opentelemetry
+mkdir -p $OPENTELEMETRY_INSTALL_DIR/lib
+echo dummy > $OPENTELEMETRY_INSTALL_DIR/lib/libopentelemetry.stub
```

Comment:
What is this `.stub` file for?

Reply:
The issue is that `vcpkg` _requires_ us to have _something_ in its own workspace lib dir. I added some temporary stub there, but I think we need to eventually post the actual `libopentelemetry_common.a`, etc. - all other libs. This is draft at this point, just to make sure that `vcpkg` compiles, builds and installs successfully. I will incrementally improve it later on, removing the `*.stub` file.

Reply:
Will address in more proper vcpkg support / follow-up PR that will publish the `opentelemetry*.a` files to install dir